src/pudl/metadata/resources/eia861.py

```diff
@@ -694,6 +694,47 @@
         "sources": ["eia861"],
         "etl_group": "eia861",
     },
+    "core_eia861__yearly_short_form": {
+        "description": (
+            """This is data extracted from Form EIA-861S, a shorter version of Form EIA-861.
+Form EIA-861S is available for certain utilities in seven out of every eight years.
```

Comment:
What do you mean by "available for certain utilities in seven out of every eight years"?

Reply:
ohhhh I see, this is about missing the 2019 data right? Might be better to just explain that the data is missing for 2019 because it's reported as part of the other tables.